backend/modules/parsers/unstructured_io.py

```diff
@@ -87,6 +87,8 @@ async def get_chunks(self, filepath: str, metadata: dict, **kwargs):
             parsed_data = response.json()
             for payload in parsed_data:
                 text = payload["text"]
+                if text == None or len(text) == 0:
```

Comment:
```suggestion
                if not text:
```

This covers both the conditions you added.

Reply:
Cool, I am just vary of how negation would turn out in cases. But can do.